### DISCUSSION TITLE
Is there a step by step available for Windows and iphone

### DISCUSSION THREAD

**Discussion by Slingboxforever** [GerryDazoo/Slinger#120] [General]
Hi All,

longtime user of slingbox 350 & 500.   Would like to keep in going.   Can someone share a step by step document I can start the transition?  I've saved the admin passwords, but need A LOT guidance after that.   :)   Thanks to anyone/everyone who can share the process.
thx

**Comment by GerryDazoo** [comment]
Well you can start by reading the readme and the release notes. That's all most people need to get going.
Put your password into the config.ini and run slingbox_server.exe. There's an excellent chance it will just work.

**Comment by f520k510** [comment]
I tried to run slinger recently on a win10 pc with python, mainly because I needed to run a beta version of slinger, which is only available in .py script and I think I will briefly go through here step by step.
0.  Download slinger from github.  Put it in a folder.
1.  You need to update admin password, ip address of slingbox, videosource, as well as port in the config.ini  I prefer using wan ip of the slingbox.
2.  Install python.  Downloadable from python.org  
3.  at a command prompt, type python -m pip install flask
and python -m install netifaces
you may encounter error that vc+ 14.0 is required when installing netifaces like I did.  I then downloaded https://visualstudio.microsoft.com/visual-cpp-build-tools and select to download desktop development with c++.  It is a 6.95gb download.  After installing, try to install netifaces again.  It should work.
4.  make sure port forward is working for your slingbox
5.  at a command prompt, type python path_to_slinger\slingbox_server.py
6.  view it with vlc or potplayer at http://ip:server port/slingbox
I find ip by go to a command prompt and type ipconfig.  I use its internal ip here as that way I don't need to do the port forward for the server ip.

If you use the exe version of the software, then it is much easier.  You only need the config.ini updated. Make sure port forward is working for your slingbox. Run slingbox_server.exe and add an exception to your firewall for the folder that containing slinger.  Then start viewing.

**Reply by GerryDazoo** [reply]
I think you can bypass the whole Visual Studio stuff if you install the "wheel" module   -m pip install wheel.
